recovery/src/main/java/com/kin/ecosystem/recovery/backup/view/BackupActivity.java

```diff
@@ -0,0 +1,88 @@
+package com.kin.ecosystem.recovery.backup.view;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.View;
+import android.view.View.OnClickListener;
+import com.kin.ecosystem.recovery.R;
+import com.kin.ecosystem.recovery.backup.presenter.BackupPresenter;
+import com.kin.ecosystem.recovery.backup.presenter.BackupPresenterImpl;
+import com.kin.ecosystem.recovery.base.BaseToolbarActivity;
+
+public class BackupActivity extends BaseToolbarActivity implements BackupView {
+
+	private BackupPresenter backupPresenter;
+
+	@Override
+	protected int getContentLayout() {
+		return R.layout.kinrecovery_frgment_activity;
+	}
+
+	@Override
+	protected void onCreate(@Nullable Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setNavigationClickListener(new OnClickListener() {
+			@Override
+			public void onClick(View v) {
+				backupPresenter.onBackClicked();
+			}
+		});
+		backupPresenter = new BackupPresenterImpl();
+		backupPresenter.onAttach(this);
```

Comment:
if the activity recreated user will be returned to startBackupFlow as backupPresenter.onAttach(this); doesn't persist current step between configuration changes